v3/linworks/fave60/fave60.json

```diff
@@ -0,0 +1,111 @@
+{
+    "name":"Linworks Fave 60",
+    "vendorId":"0x4C58",
+    "productId":"0x000D",
+    "keycodes": ["qmk_lighting"],
+    "menus": 
+            [
+              {
+                "label": "Lighting",
+                "content": [
+                  {
+                    "label": "Underglow",
+                    "content": [
+                      {
+                        "label": "Brightness",
+                        "type": "range",
+                        "options": [0, 255],
+                        "content": ["id_qmk_rgblight_brightness", 2, 1]
+                      },
+                      {
+                        "label": "Effect",
+                        "type": "dropdown",
+                        "content": ["id_qmk_rgblight_effect", 2, 2],
+                        "options": [
```

Comment:
![image](https://github.com/the-via/keyboards/assets/17491233/c174e63b-a0a7-4253-ac1e-7c346da6f3cf)

Should remove the ones that are not on this list (keep `All Off` and `Solid Color`)

So remove breathing, christmas, gradient, rgb test, alternate